cli/src/pcluster/models/imagebuilder.py

```diff
@@ -179,15 +185,25 @@ def _register_validators(self):
         self._add_validator(
             EbsVolumeTypeSizeValidator, priority=10, volume_type=Param("gp2"), volume_size=self.image.root_volume.size
         )
+        self._add_validator(
+            AMIVolumeSizeValidator,
+            priority=9,
+            volume_size=self.image.root_volume.size,
+            image=self.build.parent_image,
+        )
+        self.image.validate()
+        self.build.validate()
+        if self.dev_settings:
+            self.dev_settings.validate()
 
     def _set_default(self):
         # set default root volume
         if self.image.root_volume is None or self.image.root_volume.size.value is None:
-            increase_volume_size = 15
-            ami_id = utils.get_ami_id(self.build.parent_image.value)
-            ami_info = utils.get_info_for_amis([ami_id])
+            reserved_size = 15
```

Comment:
let's put this in a constant in the `imagebuilder.py` for now, so it can be used by the validator

Reply:
Done